# Copyright 2022 Ellis Ratner (eratner@berkeley.edu)

geometry_msgs/Pose[] desired_pose
geometry_msgs/Pose[] actual_pose